src/monero_ux_nbgl.c

```diff
@@ -248,13 +255,13 @@ static void continue_display_last(int token, unsigned char index) {
 
     fill_amount_and_destination();
     transactionContext.infoLongPress.icon = &C_Monero_64px;
-    transactionContext.infoLongPress.longPressText = "Approve";
+    transactionContext.infoLongPress.longPressText = "Hold to sign";
     transactionContext.infoLongPress.longPressToken = 0;
     transactionContext.infoLongPress.tuneId = TUNE_TAP_CASUAL;
-    transactionContext.infoLongPress.text = "Hold to confirm";
+    transactionContext.infoLongPress.text = "Sign transaction";
 
     nbgl_useCaseStaticReview(&transactionContext.tagValueList, &transactionContext.infoLongPress,
-                             "Cancel", ui_menu_validation_action);
+                             "Reject transaction", ui_menu_validation_action);
```

Comment:
Isn't it possible to simplify and `use nbgl_useCaseReview` instead of `nbgl_useCaseStaticReview`?
It would simplify the configuration...

Reply:
sadly this flow is split in several parts, so it's not possible to use the new `nbgl_useCaseReview `
refer to `ui_menu_validation_action`